digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MassTransitStopService.scala

```diff
@@ -86,24 +86,33 @@ trait MassTransitStopService extends PointAssetOperations {
     }
   }
 
+  /**
+    * Run enriching if the stop is found in Tierekisteri. Return enriched stop with a boolean flag for errors (not found in TR)
+    * @param persistedStop Optional mass transit stop
+    * @return Enriched stop with a boolean flag for TR operation errors
+    */
+  private def enrichStopIfInTierekisteri(persistedStop: Option[PersistedMassTransitStop]) = {

```

Comment:
It's defined by what it's returning and type resolution works then automatically.